data/picks/element.fm.yaml

```diff
@@ -0,0 +1,10 @@
+---
+description: Unlimited, GPLv3 Licensed, self-hostable podcast hosting
+license: null
```

Comment:
Automation can't pick up
That this is GPLv3 so we should try to remember that later

Reply:
It was never programmed to do so, as there is no structured way to do it using the current methods. My hope was in the future we would use a JB namespace in the custom XML. But I tried to make the dataset as an example so if some were put in manually they would have a place to populate that data.

Reply:
I figured there isn't, just wanted to note that we should update this file manually at some point

Reply:
Thank you both for the attention to detail! Def agreed on moving to a proper encoding of picks in the feed with license etc.